src/pymax/mixins/message.py

```
@@ -127,10 +205,19 @@ async def _upload_video(self, video: Video) -> None | Attach:
+            file_size = len(file_bytes)
+
+            # Настройки для ClientSession
+            connector = TCPConnector(limit=0)
+            timeout = aiohttp.ClientTimeout(
+                total=900, sock_read=60
+            )  # 15 минут на видео
 
             headers = {
-                "Content-Disposition": f"attachment; filename={video.file_name}",
-                "Content-Range": f"0-{len(file_bytes) - 1}/{len(file_bytes)}",
+                "Content-Disposition": f"attachment; filename={file.file_name}",
```

Ruff (0.14.8)

Undefined name `file`

(F821)